src/homepage/Components/Impact.js

```diff
@@ -23,6 +23,7 @@ export default class Impact extends Component {
                 imgUrl: "image1.svg",
                 para1: "Know right Opportunities",
                 para2: "at right time!",
+                link: "/oppcalender"
               }}
             />
             <ImpactCard
```

Comment:
`<ImpactCard />` is repeated multiple times, can't we map over a JSON? It'll look more cleaner and easier to read and maintain.

CC: @Manvityagi